test/meta_SUITE.erl

```diff
@@ -9,10 +9,12 @@
           ]
         }]).
 
--export([init_per_suite/1]).
+-export([init_per_suite/1, end_per_suite/1]).
 
 -type config() :: [{atom(), term()}].
 
 -spec init_per_suite(config()) -> config().
 init_per_suite(Config) -> [{application, sumo_db_mysql} | Config].
 
+-spec end_per_suite(config()) -> config().
+end_per_suite(Config) -> Config.
```

Comment:
@Euen that's weird, because, if you take a look to my PR (https://github.com/inaka/sumo_db_mysql/pull/16),  it was tested with OTP 19.2, and all test suites went OK.